gapis/service/path/path.proto

```diff
@@ -167,7 +264,7 @@ message ImageInfo {
 // MapIndex is a path to a value held inside a map.
 message MapIndex {
```

Comment:
> Could this be just:

No, because the `map` is a path to a server-side object, and your suggestion uses two client-side values.